core/src/Kokkos_View.hpp

```diff
@@ -821,21 +821,11 @@ class View : public ViewTraits<DataType, Properties...> {
     };
   };
 
-#if defined(KOKKOS_ENABLE_DEBUG_BOUNDS_CHECK)
-
 #define KOKKOS_IMPL_VIEW_OPERATOR_VERIFY(ARG)             \
   View::template verify_space<                            \
       Kokkos::Impl::ActiveExecutionMemorySpace>::check(); \
   Kokkos::Impl::view_verify_operator_bounds<typename traits::memory_space> ARG;
 
-#else
-
-#define KOKKOS_IMPL_VIEW_OPERATOR_VERIFY(ARG) \
-  View::template verify_space<                \
-      Kokkos::Impl::ActiveExecutionMemorySpace>::check();
-
-#endif
-
```

Comment:
Why this change?

Reply:
I couldn't avoid the warning about the passed arguments being unused differently.